interface/repository/monster_repository.go

```diff
@@ -0,0 +1,47 @@
+package repository
+
+import (
+	"encoding/csv"
+	"os"
+
+	"github.com/Le-MaliX/ACADEMY-GO-Q42021/domain/model"
+)
+
+func openCsv() ([][]string, error) {
+	file, err := os.Open("./infrastructure/datastore/monsters.csv")
```

Comment:
You would probably want to make this filepath a bit more generic. For unit testing we don't want to work with the actual data, but with a mocked file that works specifically for the purpose of testing.

Usually, all of these function are implemented by a struct that represents the application layer, there you could have a filePath attribute that you could use here. For the first deliverable it's ok, but for the next ones we would have to find a way to make this service flexible enough to be able to work with any file.